### ISSUE TITLE
TMA Lille 1.1 depicted twice

### ISSUE THREAD

**Issue by svoop** [svoop/aipp#16]
TMA 1.1 depicted twice, even though only 1 should be shown according to AIP France (highlighted part is not correct)

<img width="624" alt="Screen Shot 2019-08-10 at 21 31 15" src="https://user-images.githubusercontent.com/7255/62826097-350d0b00-bbb6-11e9-8722-61f2dd696189.png">

Reported by Stijn

**Comment by svoop** [comment]
This object present only once in the generated OFMX and thus appears to be an issue of the OFM Editor.

**Comment by svoop** [comment]
Old data, won't fix.
